app/templates/components/modals/tax-info-modal.hbs

```diff
@@ -1,7 +1,7 @@
 <div class="header">
   {{t 'Add tax information'}}
 </div>
-<div class="y-scrollable scrollable-modal content">
+<div class="modal content">
```

Comment:
@shreyanshdwivedi @uds5501 Need a small help, I was trying to use
```
<div class="{{if tax.shouldSendInvoice 'y-scrollable scrollable-' 'modal'}} content">
```
But this thing is not working and creating an extra overflow, Any method to tackle it

Reply:
@kushthedude I am curious as to how did you increased the size of the modal.

Reply:
@uds5501 I didn't increase the size, I just switched off scrolling but its not valid fix , thats why asking if the code i have written above can be applied with some changes ?

Reply:
@kushthedude should this code be using `data.event.tax.shouldSendInvoice` ?

Reply:
Since i am already under `Tax modal` , So data.event is not needed here .

Reply:
@shreyanshdwivedi @mrsaicharan1 any idea ?